.gitignore

```diff
@@ -25,6 +25,7 @@ nosetests.xml
 
 # Virtual environment
 **/venv/
+**/env/
```

Comment:
Unless there is good reason to distinguish between `venv` and `env`, can we avoid adding this?